### PR TITLE
[feat]:adding Landing Page UI

### PR DESCRIPTION
**Pull request by ultronjs**
Made landing page UI

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  padding: 1.5rem;
+  height: 100vh;
+  width: 100vw;
+  box-sizing: border-box;
+}
+.landing_page_container_left{
+  /* flex flex-col flex-jc-space-around gap-m */
```
Comment (sushil-mod) [inline src/index.css]: remove this comment if not necessary

**Review by sushil-mod — APPROVED** [review]
i found nothing as mistake , good short code , I liked your folder structure

**Review by Nagashree01 — APPROVED** [review]
The UI looks wonderful. The folder structure is good and code is readable. Good to merge. I liked how you made the routes first.

File: src/index.css (hunk 1 above)
Comment (ultronjs) [reply src/index.css]: Done

**Comment by ultronjs** [comment]
Merging after 2 Approval

**Pull request merged**